src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+/**
+ * Todo Class that is a subclass of task, no additional attributes
+ */
+public class Todo extends Task {
+
+    /**
+     * Constructor for Todo class
+     *
+     * @param taskName the details/name of the task
+     */
+    public Todo(String taskName) {
+        super(taskName);
+    }
+
+    /**
+     * Method to print the Todo task out, overrides the method in the superclass
+     */
+    @Override
+    public void printTask() {
+        System.out.print("[T]");
+        if (this.getDone()) {
+            System.out.println("[X] " + this.getTaskName());
+        } else {
+            System.out.println("[ ] " + this.getTaskName());
+        }
```

Comment:
Shouldn't you call your toString() method as the code to format the String to the output is already done there?
```suggestion
        System.out.println(toString());
```